java/src/org/dartlang/vm/service/element/Isolate.java

```diff
@@ -59,6 +59,12 @@ public ExceptionPauseMode getExceptionPauseMode() {
   }
 
   /**
+   * The list of service extension RPCs that are registered for this isolate.
+   */
+  public List<String> getExtensionRPCs() {
```

Comment:
Missing return. Looks like the java codegen has a bug. I'll look at it when I can.